### PR TITLE
Integration of bootswatch themes for bootstrap

### PR DESCRIPTION
**Pull request by alvinBM**
In this PL, I replaced the old themes with [bootswatch](https://bootswatch.com/) themes which are adapted and easily customizable made for bootstrap.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -41,7 +41,8 @@
         "league/commonmark": "^2.4",
         "symfony/dotenv": "^4.3 || 5.4",
         "twbs/bootstrap": "^5.3",
-        "twbs/bootstrap-icons": "^1.11"
+        "twbs/bootstrap-icons": "^1.11",
+        "thomaspark/bootswatch": "^5.3"
```
Comment (marclaporte) [inline composer.json:45]: FYI: https://github.com/thomaspark/bootswatch/issues/1296

**Comment by marclaporte** [comment]
This is a blocker to release Cypht 2.0: https://github.com/cypht-org/cypht/issues/879

File: composer.json (hunk 1 above)
Comment (marclaporte) [reply composer.json:45]: Keeping it alphabetical would be better

**Comment by alvinBM** [comment]
> This is a blocker to release Cypht 2.0: #879

The PL is ready for review

**Pull request merged**